gratipay/utils/i18n.py

```diff
@@ -103,17 +104,19 @@ def get_function_from_rule(rule):
 
 
 def get_text(loc, s, *a, **kw):
+    escape = kw.pop('escape', escape_noop)
```

Comment:
I'm not sure how to avoid a noop default. In https://github.com/gratipay/gratipay.com/commit/1394e2da0afe8566be8e5593d5fab7c5c4295e45 I moved the default from inside `[n_]get_text` into `add_helpers_to_context` (originally it was in `set_up_i18n`).